roles/config-docker-compose/README.md

```diff
@@ -0,0 +1,56 @@
+config-docker-compose
+=====================
+
+The config-docker-compose role is associated with installing the docker compose packages.
+
+Requirements
+------------
+
+N/A
+
+
+Role Variables
+--------------
+
+Defaults file has example of variables that can be used.
+
+| Variable | Description | Required | Defaults |
+|:--------:|:-----------:|:--------:|:--------:|
+|**docker_compose_packages**|  Packages required for docker compose. | yes | in defaults file |
```

Comment:
`in defaults file` is a bit cryptic. Please either list here, or otherwise provide the link to the entries in the defaults file.

Reply:
Also, if defined in the defaults file, the `Required` column should probably say "no" as it's not a required input from the user.

Reply:
Changed as requested.